### DISCUSSION TITLE
Does the parser need tagged data ?

### DISCUSSION THREAD

**Discussion by XavBeckers** [explosion/spaCy#9647] [Help: Other Questions] [answered]
Hello, 

After reading various pages of your documentation, it is not clear to me whether the parser requires the tagger to annotate the data first. Intuition would tell me that it does, but I don't remember reading it clearly anywhere. 
Also, it's not clear to me if the tagger also uses external data (like a list of POS-annotated vocabulary) or if it's just a model trained on data. 

Could you clarify this for me ? 

Thanks in advance.

**Comment by adrianeboyd** [comment, marked as answer]
It depends on the exact `tok2vec` configuration, but by default the parser does not rely on any other information beyond the text and the dependency annotation from the training data (and `tok2vec` may also include static vectors). The parsers in the trained pipelines from https://spacy.io/models don't rely on any tag features. You can run just the parser with the components `["tok2vec", "parser"]` in your pipeline.

**Reply by XavBeckers** [reply]
In my case, I'm working with fr_dep_news_trf that includes transformers, morphologizer, parser, attribute_ruler and lemmatizer. I presume that the transformers module do the same job than tok2vec. 
The predictions of the parser are then completely independent of the POS status of the tokens?

**Reply by adrianeboyd** [reply]
Yes, then you need `["transformer", "parser"]`.
